src/components/BookForm.jsx

```diff
@@ -80,11 +72,12 @@ function BookForm({ url }) {
       </div>
       <div className="row">
         <div className="col-12">
-          <h2 className="text-secondary my-3 text-uppercase">Add New Book</h2>
-          <form>
+          <hr className="py-2" />
+          <h2 className="add-tittle">ADD NEW BOOK</h2>
+          <form className="col-form-label-lg add-book">
             <label className="px-3" htmlFor="title">
               <input
-                className="px-2 py-1 rounded border border-secondary"
+                className="px-2 py-2 rounded border border-secondary input-title"
```

Comment:
- [ ] Great job working on the required changes. There is one more change to make and we would be ready to proceed. Your design is supposed to match the given Zeplin design as indicated by the previous reviewer. See screenshots below:
**Your design**
![image](https://github.com/PabloBona/bookstore-react-app/assets/62849872/99b5054b-21e2-47ae-8f54-e4d5154a51c9)
**Zeplin design**
![image](https://github.com/PabloBona/bookstore-react-app/assets/62849872/588d3a5f-aad2-4459-a6b9-9010ec1cc6ae)
When you look closely at the Zeplin design, you will notice that:
- `ADD NEW BOOK` text is aligned with the form element for `Book title`.
- `ADD NEW BOOK` text is aligned with the section above it where there is information about a book.
- The length of the `Category` form is quite long and the `ADD BOOK` button is aligned to the `right` with the section above that has information about a book.
Kindly effect these changes to your design

Reply:
Hi @t-yanick Is this the wrong alignment in my project? (marked in red?)
![image](https://github.com/PabloBona/bookstore-react-app/assets/110569486/a0e4407a-a3df-4e2e-ba23-f993fee4473a)

Reply:
@t-yanick are  you there ?  Can you plz guide me ?

Reply:
Hi, I think marked in red is the correct alignment.  The screenshot I added to the review is what I got running your project locally.  If the alignment marked in red is what you have now, kindly request for re-review.  Thanks